Ascend.java

```diff
@@ -2,6 +2,22 @@
 // and prints them in increasing order.

```

Comment:
OVERALL: 
largely this is a good submission. Great logic. 
From the way you wrote some of your code I guess you have personal experience with Python\JS.
I want you to see the styling guidelines for Java, document is on Moodle. 
The second thing is you shouldn't delete code given by us.
 
Commit names should be improved as well